ror-demo-cluster/conf/es/readonlyrest.yml

```
@@ -1,63 +1,199 @@
+  - username: "*"
+    groups: 
+    - local_group:
+        id: "admins"
+        name: "Platform Admins"
+      external_group_ids: [ "ELASTICSEARCH_Admins"]
+    ldap_auth: # tested, works OK
+      name: "ldap1"
+      groups: [ "ELASTICSEARCH_Admins" ]
+
+  impersonation: # tested, works OK
+    - impersonator: cen8
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)